cross-project-tests/debuginfo-tests/dexter/dex/command/commands/DexStepFunction.py

```diff
@@ -0,0 +1,38 @@
+# DExTer : Debugging Experience Tester
+# ~~~~~~   ~         ~~         ~   ~~
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""A Command that tells dexter to set a function breakpoint and step through
+the function after hitting it.
+
+NOTE: Only supported for DAP based debuggers.
+"""
+
+from dex.command.CommandBase import CommandBase
+
+
+class DexStepFunction(CommandBase):
+    def __init__(self, *args, **kwargs):
+        if len(args) < 1:
+            raise TypeError("expected 1 positional argument")
+        self.expression = str(args[0]) # Function name.
```

Comment:
Maybe name this `self.function` instead?

Reply:
I think there was a reason I used self.expression. But I don't recall why (it was a while ago, while I was making the stepping-graphs for key instructions. It could just have easily been a mistaken assumption). I'm happy to change it to self.function and if we run into issues (perhaps with printing?) we can reassess, rather than have unnecessarily cryptic code.